acid-view/app/Main.hs

```diff
@@ -0,0 +1,21 @@
+module Main where
+
+import AcidView (loadState, printDB)
+import CmdParse (CmdOptions(..), parsedCMDOptions)
+
+import Ariadne.Wallet.Cardano.Kernel.DB.AcidState (Snapshot(..), defDB)
+import Data.Acid (query)
+
+main :: IO ()
+main = do
+  CmdOptions{..} <- parsedCMDOptions
+  stateE <- loadState path defDB position
+  case stateE of
+    Left err -> print err

```

Comment:
`err` is already a string, use `putStrLn`

Reply:
thanks